docs/quickstart.md

```diff
@@ -379,3 +379,25 @@ value to be more or less strict:
 ```python
 >>> httpx.get('https://github.com/', timeout=0.001)
 ```
+
+## Authentication
+
+HTTPX supports Basic and Digest HTTP authentication.
+
+To provide Basic authentication credentials you can simply pass a 2-tuple of
```

Comment:
I remember someone recently revamped the Django documentation to remove all instances of "simply", "just", "basically" etc. Let's also not assume things are simple for our users. :)

```suggestion
To provide Basic authentication credentials, pass a 2-tuple of
```